sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_administration_client.hpp

```diff
@@ -44,57 +44,31 @@ namespace Azure { namespace Security { namespace Attestation {
    *
    */
   class AttestationAdministrationClient final {
-
   public:
-    /**
-     * @brief Destructor.
-     *
-     */
-    virtual ~AttestationAdministrationClient() = default;
-
-    /**
-     * @brief Construct a new Attestation Administration Client object.
-     *
-     * @param endpoint The URL address where the client will send the requests to.
-     * @param credential The authentication token to use.
-     * @param options The options to customize the client behavior.
-     */
-    explicit AttestationAdministrationClient(
-        std::string const& endpoint,
-        std::shared_ptr<Core::Credentials::TokenCredential const> credential,
-        AttestationAdministrationClientOptions const& options
-        = AttestationAdministrationClientOptions());
-
-    /**
-     * @brief Returns the API version the client was configured with.
-     *
-     * @returns The API version used when communicating with the attestation service.
-     */
-    std::string const& ClientVersion() const { return m_apiVersion; }
-
     /**
      * @brief Construct a new Attestation Administration Client object from another attestation
      * administration client.
      *
      * @param attestationClient An existing attestation client.
      */
-    explicit AttestationAdministrationClient(
+    AttestationAdministrationClient(
```

Comment:
this is still public right?  so do we want to keep the `explicit` ?

Reply:
It's not public any more. And it can't be explicit otherwise you can't return a std::unique_ptr to it :(.